plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/MavenPathParser.java

```diff
@@ -10,7 +10,7 @@
  * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
  * Eclipse Foundation. All other trademarks are the property of their respective owners.
  */
-package org.sonatype.nexus.repository.maven.internal;
+package org.sonatype.nexus.repository.maven;
```

Comment:
Yeah, if @mcculls can offer some guidance on this, but it's the tests that should accommodate, not the production code IMHO